cs3client/file.py

```diff
@@ -86,7 +86,7 @@ def set_xattr(self, auth_token: tuple, resource: Resource, key: str, value: str,
         :raises: UnknownException (Unknown error)
         """
         md = cs3spr.ArbitraryMetadata()
-        md.metadata.update({key: value})  # pylint: disable=no-member
+        md.metadata.update({key: str(value)})  # pylint: disable=no-member
```

Comment:
If I remember correctly, the idea was to let the user convert the value to a string before providing it after Diogo and I discussed it. However, I don't see any problem with doing it this way.

Reply:
Hi there, the review is much appreciated!

And for this change, I remember now the discussion, but the error I got without the conversion to `str` seemed to me pretty much CS3API-dependent, not related to the wopiserver. So I guess it makes sense to do it here, @diocas what do you think?

Reply:
Since `value` is of type `str` (method parameter), I would expect Python to have raised an exception when you called the function without a string. So, converting str to str again seems unnecessary.

Reply:
I'd have to check which kind of exception I got, anyway fine, I reverted that and moved the `str()` conversion on the wopiserver side. To be seen if this can ever be used with other types (I remember the CS3API itself did not allow that).